lib/help/debug.txt

```diff
@@ -29,7 +29,7 @@ will not be scored if you use debug commands.
  *****debug.txt*29[o  Edit object attributes]               O  (unused)
  *****debug.txt*31[p  Phase door]                           P  (unused)
  *****debug.txt*33[q  Get a quest]                          Q  (unused)
- r  (unused)                             *****debug.txt*36[R  Create a trap]
```

Comment:
Isn't there also some mention of the (D)isarm command in the regular command listing?